### DISCUSSION TITLE
<title>PLANET

### DISCUSSION THREAD

**Discussion by moncefdhk999** [community/community#40313] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

POURQUOI TU M ENVOUYER EN FRACAIS JE NE COMPANDRE QUE QUELQUE MOST CEST TOUS  EN ANGLAIS JE NE PEUX PAS TOUTE SEUL JE BEACOUP DES PROBLEM SUR NET ALL

**Comment by ossanna16** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
